logexporter/cmd/main.go

```diff
@@ -44,6 +45,9 @@ var (
 	enableHollowNodeLogs = flag.Bool("enable-hollow-node-logs", false, "Enable uploading hollow node logs too. Relevant only for kubemark nodes")
 	sleepDuration        = flag.Duration("sleep-duration", 60*time.Second, "Duration to sleep before exiting with success. Useful for making pods schedule with hard anti-affinity when run as a job on a k8s cluster")
 	dumpSystemdJournal   = flag.Bool("dump-systemd-journal", false, "Whether to dump the full systemd journal")
+
+	// NOTE(oxddr): cl2-* is prefix we expect from files created by clusterloader2
+	extraLogFiles = pflag.StringSlice("extra-log-files", []string{"cl2-*"}, "Extra log files to dump")
```

Comment:
I'm fine with adding a flag, but I don't think we should default it to cl2 - if we really want that, this should be hidden somewhere below when we call logexporter for scalability jobs...

Reply:
Removed. This needs to be set in log-dump script in k/k - I'll send a separate PR.